app/config/question-bank.js

```diff
@@ -2814,7 +2814,7 @@ const questionBank = {
             </div>`,
             messageLink: {
               url: 'https://www.gov.uk/government/collections/rural-payments-and-grants',
-              title: 'See other grants you may be eligible for.'
+              title: 'See other grants you may be eligible for'
```

Comment:
This is TBC as all the other applications seem to have it. As a result, i think this can be left as **See other grants you may be eligible for.** for now :)